Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -97,17 +90,11 @@ struct ILBMLoadingContext {
     // max number of bytes per plane row
     u16 pitch;
 
-    // Current file color_table in BGRA8888 format
-    Vector<u32> color_table;
-
-    ByteBuffer pixel_data;
+    FixedArray<Color> color_table;
 
     RefPtr<Gfx::Bitmap> bitmap;
 
-    // BMHD IFF-ILBM header
-    BitmapHeader bm_header;
-
-    Optional<IntSize> size;
+    BitmapHeader BMHDHeader;
```

Comment:
I was more thinking of `BMHDHeader bm_header;`
Variable names shouldn't be capitalized